packages/core/release/package.json

```diff
@@ -19,14 +19,16 @@
     "bin",
     "lib"
   ],
+  "peerDependencies": {
+    "conventional-changelog-angular": "^6.0.0"
+  },
```

Comment:
Moved this to a peer dependency, since it needs to be installed at the top-level.